components/menu/menu-item-mixin.js

```diff
@@ -8,6 +8,7 @@ export const MenuItemMixin = superclass => class extends superclass {
 			hidden: { type: Boolean, reflect: true },
 			last: { type: String, reflect: true }, // set by d2l-menu
 			role: { type: String, reflect: true },
+			tabindex: { type: String, reflect: true },
 			text: String
 		};
 	}
```

Comment:
Nit: the keycodes below could be moved out of the class.

Reply:
They are also used in `menu-item-link.js` (https://github.com/BrightspaceUI/core/blob/master/components/menu/menu-item-link.js#L74) so I think they would need to stay as is